app/locale.js

```diff
@@ -140,7 +140,7 @@ exports.translation = function (token) {
     // This will return an identifier in upper case enclosed in square brackets
     // Useful for determining if a translation was not requested in the menu
     // identifiers above.
-    return '[' + token.toUpperCase() + ']'
+    return token.toUpperCase()
```

Comment:
Thanks for the fixes.  Note that menu will still flash FILE EDIT VIEW then go to File Edit View